tools/extract-persist.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+set -e
+
+TMPDIR=./tmp
+OUTFILE=${TMPDIR}/persist.tgz
+TMPFILE=${TMPDIR}/live.qcow2
+INFILE=./dist/amd64/live.qcow2
```

Comment:
If you have a long-running qemu device (which was installed to the target), then this needs to be the target image. I don't know who uses the installer on qemu but it might make sense to allow specifying the INFILE on the command line.

Reply:
I was a little hesitant to do that, since then you have a longish dependency to type in. This is just a single command that is easy to type and does it all.

What would `INFILE` look like for installer?

Reply:
Once you have run the installer the virtual disk used by the virtual EVE is in dist/amd64/target.img

Of course it and live are different for ARM.

Reply:
So we are talking about four different potential files:

* amd64 live: `dist/amd64/live.img`
* amd64 installer: `dist/amd64/target.img`
* arm64 live: `dist/arm64/live.img`
* arm64 installer: `dist/arm64/target.img`

Is that correct?

Reply:
Yes.

Reply:
Ok I’ll adjust

Reply:
Adjusted. You now can run 

```
extract-persist.sh <type> <arch>
```

Where:

* `type` - `live` or `installer`
* `arch`- `amd64` or `arm64`

It defaults to `amd64` and `live`, but can override simply.